core/benches/snapshot/snapshot.rs

```diff
@@ -17,8 +19,8 @@ macro_rules! fake_extensions {
           deno_core::extension!(
             $name,
             ops = [ ops::$name ],
-            esm_entry_point = concat!("ext:", stringify!($name), "/file.js"),
-            esm = [ dir "benches/snapshot", "file.js", "file2.js" ]
+            esm_entry_point = concat!("ext:", stringify!($name), "/file.ts"),
+            esm = [ dir "benches/snapshot", "file.ts", "file2.ts" ]
```

Comment:
I think we should split this into a typescript and non-typescript bench so we can see the relative impact of transpilation.